remo/voting/tasks.py

```diff
@@ -138,7 +141,7 @@ def create_rotm_poll():
                                     end__range=[start, end])
 
     if not now().date() > ROTM_NOMINATION_END_DATE or rotm_poll.exists():
```

Comment:
Because if the flag is active, we want actually to create the poll. Basically if the switch is active we want it to bypass this check.